website/docs/docs/cloud/about-cloud/architecture.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "dbt Cloud Architecture"
+id: "architecture"
+description: "Information about the architecture, communication, and security of dbt Cloud"
+---
+
+This page is intended to help practitioners and those interested in the architecture and data flow of the hosted dbt Cloud product.
+
+## About dbt Cloud architecture
+
+The dbt Cloud application comprises a set of static and dynamic components. The static components are constantly running to serve highly available dbt Cloud functionality, for example, the dbt Cloud web application. The dynamic components are created ad-hoc to fill background jobs or requests to use the IDE. 
+
+dbt Cloud is available in most regions around the world in both [single tenant](/docs/cloud/about-cloud/tenancy#single-tenant)(AWS and Azure) and [multi-tenant](/docs/cloud/about-cloud/tenancy#multi-tenant) configurations.  
+
+dbt Cloud uses PostgreSQL for its backend, S3-compatible Object Storage systems for logs and artifacts, and a Kubernetes storage solution for creating dynamic, persistent volumes. 
+
+All data at rest on dbt Cloud servers is protected using AES-256 encryption. 
```

Comment:
just double checking if it's meant to say 'at rest'?

Reply:
Yup! When the data is on our servers (vs in transit over the internet), it is considered "at rest." It's a different type of encryption than when the data is moving over the internet